slicer_cli_web/web_client/templates/fileWidget.pug

```diff
@@ -2,9 +2,16 @@ extends ./widget.pug
 
 block input
   .input-group
-    if value && value.name
+    if value && value.name && type === 'multi' && folderName !== undefined
+      - value = `${folderName}/RegEx(${value.name()})`;
+    else if value && value.name
       - value = value.name();
     - var placeholder = 'Choose a file...'
+    - var batchload = false
+    - if (['file','item','image', 'multi'].includes(type))
```

Comment:
Until we have support for inputs that expect to take multiple values (rather than batching over inputs that take single values), we should disable showing the batch options for those inputs.  This would change this line to
```
  - if (['file','item','image', 'multi'].includes(type) && !multiple)
```